core/domain/suggestion_services.py

```diff
@@ -412,6 +413,39 @@ def accept_suggestion(
         _update_user_proficiency(user_proficiency)
 
 
+def update_suggestion(suggestion_id, translation_html):
```

Comment:
This function is for a specific type of suggestion so a better name would be update_translation_suggestion*

Reply:
Done